challenges/cloud/sre-server.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func main() {
+	err := run(os.Args)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v", err)
+	}
+}
+
+func run(args []string) error {
+	homedir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+
+	var port, kubeconfig string
+	flag.StringVar(&port, "port", "8080", "server port")
+	flag.StringVar(&kubeconfig, "kubeconfig", filepath.Join(homedir, ".kube", "config"), "path to the kubeconfig file")
+	flag.Parse()
+
+	// use the current context in kubeconfig
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	// create the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	// add handlers
+	http.Handle("/healthz", &healthzHandler{clientset: clientset})
+	http.Handle("/metrics", http.NotFoundHandler()) // TODO: add metrics
```

Comment:
Is this metrics `TODO` something outstanding for us or for the candidate?  I don't see any reference to it later in the challenge document.

Reply:
@gozer, @jimbishopp  I believe we dropped metrics, and dashboarding as an ask -- is that correct?

If so we might as well remove this from the server. If not, and it's still a requirement we might be pushing the limit on what's reasonable to expect from our candidates.

Reply:
Yes, we did drop metrics and dashboarding as an ask, IIRC, however, I'd say just leave it there as it is, after all, this is supposed to be a half-made POC with cruft possibly in it. Maybe the candidate leaves it, maybe they clean it up and remove it, maybe they implement it, maybe they ask good questions about telemetry?

I don't see a big downside to leaving it there, just one more source of a potential signal IMO.

Reply:
Yeah, I think an oversight. I don't remember making a conscious decision together to remove it.

With that said, there's plenty in this challenge to help me form an opinion on a candidates skillset, so let's remove this for now.